src/components/biggive-footer/biggive-footer.tsx

```diff
@@ -203,6 +220,14 @@ export class BiggiveFooter {
                   <li>
                     <a href={makeURL('Blog', this.blogUrlPrefix, 'privacy')}>Privacy Statement</a>
                   </li>
+                  <li>
+                    <a href={makeURL('Blog', this.blogUrlPrefix, 'privacy#cookies')}>Cookies Statement</a>
+                  </li>
+                  <li>
+                    <a href="javascript:void(0);" onClick={this.emitShowCookiePreferences}>
```

Comment:
Not sure about the accessibility & UX of this. Have we checked if a screen reader can successfully use it like a non-JS link? Should we maybe consider an anchor fallback (something like a `#openCookies` that the footer itself checks for on init), so that opening in a new tab is not broken?

Reply:
I'll test with the Ubuntu screenreader. I think since it does have both an href and an onClick there's no reason for screen readers not to consider it a clickable element.

I don't see opening in a new tab as critical for this but agree it would be nice to support that. In which case maybe this shouldn't be a JS link at all, it could simply link to the hash? We can then detect that in an angular way in FE, and wordpress use vanilla JS to listen to the HashChange event ( https://developer.mozilla.org/en-US/docs/Web/API/Window/hashchange_event / https://caniuse.com/hashchange ) and get rid of the new event emitter. What do you think?

Reply:
Or maybe even better and simpler - since hashchange is not supported in opera mini - don't use a hash at all, and make a dedicated page with its own route for this in FE. It shouldn't be a problem to navigate away from whatever page the user was looking at. Ideally it might not be a modal then and just a page full of cookie options, but that's more work than keeping it in the modal we have.

So it could be either the modal with a blank background, or maybe the modal over a copy of the donate homepage. When the user closes the modal it would automatically navigate to the regular donate homepage URL.

Reply:
btw href="javascript:void(0);" is what we have already have by default on any biggive-button that doesn't have an href passed in, including the ones we use for cookie options. See comment https://github.com/thebiggive/components/blob/cf6175ea0272eac2219e87db78389df0eeb87ca8/src/components/biggive-button/biggive-button.tsx#L69

Reply:
I've checked now, the link does indeed work fine with keyboard and ears with the Orca screen reader in Ubuntu.  But I think I'll still go ahead and change it to be a simple non hash link to a page on the donate site - then we don't need to do any work to handle it in Wordpress.

Reply:
Changed to regular non-hash link in 9a5576a.